pandas/io/excel.py

```diff
@@ -312,11 +327,6 @@ def _range2cols(areas):
             >>> _range2cols('A,C,Z:AB')
             [0, 2, 25, 26, 27]
             """
-            def _excel2num(x):
-                "Convert Excel column name like 'AB' to 0-based column index"
-                return reduce(lambda s, a: s * 26 + ord(a) - ord('A') + 1,
```

Comment:
ok, opened #16510 for this functionality.  will re-submit with only the argument change (parse_cols -> usecols).